src/router.ts

```diff
@@ -0,0 +1,26 @@
+import { Router } from 'express';
+import { InvalidArgument } from './utils/errors/client.error';
+import UserController from './user/user.controller';
+import { wrapAsync } from './utils/wrappers';
+
+const appRouter = Router();
```

Comment:
In my opinion, the router file should be cleaner. The content of each wrapped function should be in a separate file, in the controller.